inputs/worker_inner.json

```diff
@@ -0,0 +1,155 @@
+{
+  "client_version": "{{VERSION}}",
+  "prebuild_plugins": [
+    {
+      "name": "add_filesystem",
+      "args": {
+        "koji_hub": "{{KOJI_HUB}}",
+        "repos": []
+      }
+    },
+    {
+      "name": "pull_base_image",
+      "args": {
+        "parent_registry": "{{REGISTRY_URI}}",
+        "parent_registry_insecure": true
+      }
+    },
+    {
+      "args": {
+        "hub": "{{KOJI_HUB}}"
+      },
+      "name": "bump_release"
```

Comment:
We should be consistent in the order of the "args" and "name" keys to make it easier to read.